get.php

```diff
@@ -28,20 +28,30 @@
 /**
  * Set include path
  */
-
+$paths = [];
 $paths[] = $bp . $ds . 'app' . $ds . 'code' . $ds . 'local';
 $paths[] = $bp . $ds . 'app' . $ds . 'code' . $ds . 'community';
 $paths[] = $bp . $ds . 'app' . $ds . 'code' . $ds . 'core';
 $paths[] = $bp . $ds . 'lib';
 
 $appPath = implode($ps, $paths);
 set_include_path($appPath . $ps . get_include_path());
-
 include_once 'Mage/Core/functions.php';
 include_once 'Varien/Autoload.php';
 
 Varien_Autoload::register();
 
+/** AUTOLOADER PATCH **/
+$autoloaderPath = getenv('COMPOSER_VENDOR_PATH');
```

Comment:
When I tested this `$autoloaderPath = getenv('COMPOSER_VENDOR_PATH');` in my OM instance that returns 404, the value of `$autoloaderPath` is `false`.  I kinda expect it to be `true` to bypass the if block. Test script added at line 46:

```php
/** AUTOLOADER PATCH **/
$autoloaderPath = getenv('COMPOSER_VENDOR_PATH');
var_dump($autoloaderPath);
exit();
```
Output:
```
bool(false)
```

If I added my test script before line 52, the output is
```
string(33) "/home/celeraes/public_html/vendor"
```
which is expected because the if block is executed. But in my case, I don't need this patch because https://celeraeshop.com/media/catalog/category/blankx.jpg returns 404.

So, why not just add the vendor path in `$paths` and do away with the check. See
https://github.com/OpenMage/magento-lts/blob/10b63c1f65bd38f4d75a1c049d9dedaf9dffddde/lib/Varien/Autoload.php#L52-L59

Lines 56-58 check if file exists and do the `require` thing, same as your code.

Reply:
do you have your webserver usign the htaccess in the media folder? until I enabled it (I use Caddy now so it doesn't support htaccess) I couldn't see the error, but the error is there.

This PR just copies the code from index.php/Mage.php since get.php is actually an entry point for OpenMage, it should have the same code as index.php